apps/api/docs/database-design.md

```diff
@@ -0,0 +1,49 @@
+# **Database Design**
+
```

Comment:
Keep the table names as `jobs` and `files`

Reply:
Table names changed to jobs and files.
 **files :** 
 **jobs :**